recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
 
 build:
     skip: True  # [win]
-    number: 0
+    number: 1
```

Comment:
Probably wasn't necessary to bump the build. Remembering that each time a change is proposed to this feedstock, the build is checked anyway, so we can continue to have confidence that we haven't broken the recipe.

Anyway, :+1: